specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/applicationGateway.json

```diff
@@ -1895,6 +1920,28 @@
       ],
       "description": "Allows to disable rules within a rule group or an entire rule group."
     },
+    "ApplicationGatewayFirewallExclusion": {
+      "properties": {
+        "matchVariable": {
+          "type": "string",
+          "description": "The variable to be excluded."
+        },
+        "selectorMatchOperator": {
+          "type": "string",
+          "description": "The operation on selector."
```

Comment:
This is hard to understand and not very helpful.  Perhaps more something like this: "Operator used to control how matchVariable matches this exclusion entry". I don't fully remember the options here, so maybe I'm mistaking.

Reply:
will change